widgetastic_manageiq/__init__.py

```diff
@@ -1038,6 +1038,28 @@ def workaround_save_issue(self):
             "{}._handlers.change.map(function(handler) {{ handler() }});".format(self.item_name))
 
 
+class SSUIlist(Widget, ClickableMixin):
```

Comment:
We should talk to mfalesni about if we want to have this in sa new module/package or not.

Reply:
@mfalesni : Please review these widgets